common/README.md

```diff
@@ -30,6 +30,18 @@ The DNS zones are:
 - dev.gcp.gpii.net
 - ${user}.dev.gcp.gpii.net
 
+The zones "gpii.net", "gcp.gpii.net", "test1.gpii.net" and "gcp.test1.gpii.net" need to be created manually before this code creates the rest of the resources.
```

Comment:
Why can't we manage these zones in code?

Reply:
We can, but we decided to not do so because the zones are managed by AWS:

https://issues.gpii.net/browse/GPII-2883?focusedCommentId=39729&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-39729